pptx_blueprint/__init__.py

```diff
@@ -42,14 +42,57 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
         """
         pass
 
-    def replace_table(self, label: str, data) -> None:
+    def replace_table(self, label: str, data, header=False, rownames=False):
         """Replaces rectangle placeholders on one or many slides.
 
         Args:
             label (str): label of the placeholder (without curly braces)
             data (pandas.DataFrame): table to be inserted into the presentation
         """
-        pass
+        assert isinstance(data, pandas.dataframe)
```

Comment:
You could remove the assertion here and include type hints in the method signature like so:

```python
def replace_table(self, label: str, data: pandas.dataframe, *, header: bool = False, rownames: bool = False) -> None:
```